app/forms/declaration/PackageInformation.scala

```diff
@@ -47,48 +54,31 @@ object PackageInformation extends DeclarationPage {
   private val NumberOfPackagesLimitLower = 0
   private val NumberOfPackagesLimitUpper = 99999
 
-  private val mappingTypesOfPackagesOptional: Mapping[Option[String]] = optional(
-    text()
-      .verifying("declaration.packageInformation.typesOfPackages.error", isContainedIn(PackageTypes.all.map(_.code)))
-  )
+  private def generateId: String = Random.alphanumeric.take(8).mkString.toLowerCase
```

Comment:
This is inconsistent with the BE way of creating Id. It is using the same logic as `ExportItemIdGeneratorService` which is not equal to what is here.

Reply:
True.  But what difference does it make?  A random string is a random string.